website/src/constants.ts

```diff
@@ -1,2 +1,23 @@
 export const FIRST_ELEMENT = 0;
 export const ONE = 1;
+export const FAQ = [
```

Comment:
please move this into the language/en.yml file

Reply:
Hi @Idrinth sure i'm fixing my branch merge issue after that i'll make the changes.

Reply:
Hi @Idrinth sorry i didn't get this part that i have to move this FAQ constant array (changing it to yml format)to en.yml and read from there?

Reply:
Texts are meant to be loaded via Lang-component, not via constants.

Reply:
Hi @Idrinth, i've added them in the yml and ran the npm run language command.So i justed wanted to know that i want to iterate it and i noticed the we create json of the yml file in public/locales/en/translation.json 

Can i use transaltion.json in my component ? Because i want to iterate over the data so that i can populate them is that the correct way. Otherwise i'm not able to get the array (refer code image).
<img width="874" alt="image" src="https://github.com/Idrinth/api-bench/assets/47591431/1ce15343-ea48-4349-9d76-1c62630b1cf7">
<img width="762" alt="image" src="https://github.com/Idrinth/api-bench/assets/47591431/ea6bb4ed-67ab-4409-aaf5-0e526687ef4a">

Reply:
you can build the key by using a slightly different structure:
``ỳml
home:
  faq:
    title: "FAQ"
    questions:
      title_1: "Wha..."
      description_1: "desc..."
```
and in the element you can just do a for loop with the amount of items currently intended.